src/steam.cpp

```diff
@@ -613,6 +613,12 @@ void steamAchievement(const char* achName)
 	return;
 #else
 
+	if ( !(svFlags & SV_FLAG_CHEATS) )
```

Comment:
The if statements need the ! removed. This implements the opposite effect, you only get Steam Achievements when cheats are on. 

For reference, when cheats are enabled, svFlags = 31 and SV_FLAG_CHEATS is constant 1. `!svFlags & SV_FLAG_CHEATS` equals 0, which skips the return label and gives the player the achievement. When cheats disabled, then svFlags = 30, `!svFlags & SV_FLAG_CHEATS` equals 1, which returns. 

`if ( !(svFlags & SV_FLAG_CHEATS) )
{
//code
}` should be 
`if ( (svFlags & SV_FLAG_CHEATS) )
{
//code
}`

Reply:
I'll note the bitwise operations are very hard to visually determine what's going on. I'll admit it looked correct before I convinced myself otherwise by doing a playtest.